projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+# Reuse places & organizers
```

Comment:
You'll notice I used both the words `place` and `location` on this documentation page. This is on purpose, since `place` is the term used in the endpoint of Search API and `location` the name of the property inside the event body that integrators send to the Entry API.